extensions/wikia/RWEPageHeader/RWEContributeMenu.class.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+class RWEContributeMenu {
+
+	private $defaultSpecialPagesData = [
+		'Upload' => [
+			'label' => 'oasis-navigation-v2-add-photo'
+		],
+		'CreatePage' => [
+			'label' => 'oasis-navigation-v2-create-page',
+			'class' => 'createpage',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "class" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)